beacon-chain/sync/initial-sync/service.go

```diff
@@ -0,0 +1,265 @@
+package initialsync
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/types"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/p2p"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.WithField("prefix", "sync")
```

Comment:
should this be "initialsync"?

`var log = logrus.WithField("prefix", "initialsync")`